swagger2.0/oic.r.vodlist.swagger.json

```diff
@@ -0,0 +1,135 @@
+    }
+  },
+  "schemes": ["http"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/VODListResURI" : {
+      "get": {
+        "description": "This resource describes the VODs that have been onboarded on the Bridge Platform.\n",
+        "parameters": [
+          {"$ref": "#/parameters/interface"}
+        ],
+        "responses": {
+            "200": {
+              "description" : "",
+              "x-example":
+                {
+                  "rt":    ["oic.r.vodlist"],
+                  "vods":  [
+                    {
+                      "n": "Window shatter sensor",
+                      "di": "54919CA5-4101-4AE4-595B-353C51AA1234",
+                      "econame": "Z-Wave"
+                    },
+                    {
+                      "n": "Door Lock",
+                      "di": "54919CA5-4101-4AE4-595B-353C51AA5678",
+                      "econame": "Zigbee"
+                    }
+                  ]
+                },
+              "schema": { "$ref": "#/definitions/vodlist" }
+            }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "interface" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.r", "oic.if.baseline"]
+    }
+  },
+  "definitions": {
+    "vodentry" : {
+      "description": "Information of a VOD created by the Bridge",
+      "type": "object",
+      "properties": {
+        "n": {
+            "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/n"
+        },
+        "di" : {
+          "description": "Device Id of VOD",
+          "allOf": [
+            {
+              "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.types-schema.json#/definitions/uuid"
+            },
+            {
+              "description": "Unique identifier for device",
+              "readOnly": true
+            }
+          ]
+        },
+        "econame": {
+          "description": "Ecosystem Name of the Bridged Device which is exposed by this VOD",
+          "type": "string",
+          "enum": [ "BLE", "oneM2M", "U+", "Zigbee", "Z-Wave" ],
+          "readOnly": true
+        }
+      },
+      "required": ["n", "di", "econame"]
+    },
```

Comment:
If "n" is blank for the VOD itself, is the intent that this Resource would expose "n": "" (i.e. a blank Property Value)?

Reply:
Yes.  'n' is mandatory for /oic/d so it has to be there.

Reply:
So we'll need spec text that ties "n" of /oic/d to "n" of this Resource.

Reply:
Agreed. Added details to the description also.  Please check.

Reply:
Looks good.